tests/test_noise_models.py

```diff
@@ -25,22 +25,33 @@
 
 def add_DM_noise_to_model(model):
     all_components = Component.component_types
-    noise_class = all_components["PLDMNoise"]
-    noise = noise_class()  # Make the DM noise instance.
-    model.add_component(noise, validate=False)
+    model.add_component(all_components["PLDMNoise"](), validate=False)
     model["TNDMAMP"].quantity = 1e-13
     model["TNDMGAM"].quantity = 1.2
     model["TNDMC"].value = 30
     model.validate()
-    return model
+
+
+def add_chrom_noise_to_model(model):
+    all_components = Component.component_types
+    model.add_component(all_components["PLChromNoise"](), validate=False)
+    model["TNCHROMAMP"].quantity = 1e-14
+    model["TNCHROMGAM"].quantity = 1.2
+    model["TNCHROMC"].value = 30
+
+    model.add_component(all_components["ChromaticCM"](), validate=False)
+    model["TNCHROMIDX"].value = 4
+
+    model.validate()
 
 
 @pytest.fixture()
 def model_and_toas():
     parfile = examplefile("B1855+09_NANOGrav_9yv1.gls.par")
     timfile = examplefile("B1855+09_NANOGrav_9yv1.tim")
     model, toas = get_model_and_toas(parfile, timfile)
-    model = add_DM_noise_to_model(model)
+    add_DM_noise_to_model(model)
+    add_chrom_noise_to_model(model)
```

Comment:
you are adding both types to the same model?  Is that what you want to do?  You don't want to add to two separate models and compare?

Reply:
Yes. The tests in `test_noise_models.py` check things like the shape of weight and basis matrices etc. It does not do any "science" tests. So I think it's safe to piggyback like this. 

Things like simulation and fitting tests are in `test_plchromnoise.py`.